pkg/ebpf/httpfltr/httpfltr_linux.go

```diff
@@ -48,3 +49,32 @@ func findNamespace(pid int32) (uint32, error) {
 
 	return 0, fmt.Errorf("couldn't find ns pid in the symlink [%s]", nsPid)
 }
+
+func findSharedLib(lib string) (string, error) {
+	o, err := exec.Command("ldconfig", "-p").Output()
```

Comment:
I really have no idea of a better way to do this. I thought about (and tried) using loading of the shared library with unsafe and C linkage, but it was very hacky and messy. This works, but I'm not sure if it's possible to have a distro without ldconfig. I'm really looking for suggestions :).

Reply:
I should mention that libbpf doesn't have this problem with the path, as long as the library is in the standard location, declaring:

```
SEC("uprobe/libssl.so:SSL_read")
```

is enough for it to resolve the library. 

With cilium however, I wasn't sure how to make this happen, their uprobe examples use a harcoded path.

Reply:
The `busybox:1.36.1-glibc` docker image doesn't bundle `ldconfig`, we should install it in the `Dockerfile` definition.

Reply:
Argh, that's what I was afraid of. I'm going to merge this, but look for a more reliable way to find the path. Thanks for checking.